src/referencePackages/src/microsoft.codeanalysis.csharp.workspaces/4.0.1/Customizations.props

```diff
@@ -0,0 +1,8 @@
+<Project>
+
+  <ItemGroup>
+    <!-- Humanizer is provided through source-build-externals -->
+    <ProjectReference Remove="../../humanizer.core/2.2.0/Humanizer.Core.2.2.0.csproj" />
```

Comment:
Would it make sense to replace this with a package dependency instead?  The primary benefit is to ensure the dependency exists to know that the resulting nuget package can be restored without prebuilts.  This applied to all places using this pattern.